lib/devise_saml_authenticatable/competitions_attribute_map_resolver.rb

```diff
@@ -0,0 +1,7 @@
+module DeviseSamlAuthenticatable
+  class CompetitionsAttributeMapResolver < DeviseSamlAuthenticatable::DefaultAttributeMapResolver
+    def attribute_map
+      COMPETITIONS_CONFIG[:saml][:attribute_map]
+    end
```

Comment:
I suggest that you add a unit test for this hack to make sure upgrading the gem doesn't break it.

Reply:
sorry I thought this was the hack John was talking about. I was confused. This is fine. I think integration test can cover this. Ignore my previous comment.